request-handler/src/main/java/au/org/aodn/aws/wps/lambda/WpsLambdaRequestHandler.java

```diff
@@ -34,26 +34,27 @@ public AwsApiResponse handleRequest(AwsApiRequest request, Context context) {
         try
         {
             String envName = getEnvironmentName(context);
```

Comment:
This and the code below looks specific to getting config using aliases etc so perhaps better located with code specific to that?